pepys_import/file/importer.py

```diff
@@ -45,19 +45,18 @@ def can_load_this_header(self, header) -> bool:
     @abstractmethod
     def can_load_this_file(self, file_contents) -> bool:
         """Whether this parser can handle this whole file
-        
+
         :param file_contents: Whole file contents
         :type file_contents: String
         :return: Yes/No
         :rtype: bool
         """
 
-    def init_load_this_file(self, data_store, path, file_object, datafile):
```

Comment:
@robintw - I'm not fully happy with the name `core_load_file_file`, but I think this implementation simplifies the file_processor logic.

Reply:
Yes, I couldn't work out what to call it either. Probably just stick with `core_load_this_file` until we can come up with a better name.

Reply:
I _think_ I'm following the logic here: https://stackoverflow.com/a/34225912/92441